api/tests/core/providers/test_api.py

```diff
@@ -287,10 +289,15 @@ def test_build_new_offers_from_stock_details(self):
             provider_id=provider.id,
         )
 
+        # We need to commit to have the offer description property working
+        db.session.add_all([*new_offers, product])
+        db.session.commit()
+
         # Then
         assert len(new_offers) == 1
         new_offer = new_offers[0]
         assert new_offer.bookingEmail == "booking_email"
+        assert new_offer._description == None
```

Comment:
Détail (par homogénéité dans le code PC), ici et pour ceux en-dessous : 
```suggestion
        assert new_offer._description is None
```